velox/expression/tests/SimpleFunctionPrestNullsTest.cpp

```diff
@@ -26,6 +27,11 @@
 #include "velox/vector/ComplexVector.h"
 #include "velox/vector/SelectivityVector.h"
 
+// This file have tests that ensure that simple function adapter, does properly
```

Comment:
Looks like there is a typo in the file and test name: "Prest" in SimpleFunctionPrestNullsTest.cpp

Some typos in this comment:

This file contains tests to ensure that simple function adapter properly sets ... It was added after we discovered that simple function adapter ...

Reply:
let me change the file name.